docs/conf.py

```diff
@@ -68,9 +68,12 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
nit: I like the alabaster theme more than this one 😄 
Is it possible we could use alabaster for our docs with readthedocs?